app/middleware/gp.js

```diff
@@ -83,9 +84,40 @@ function getUrl(req, res, next) {
   next();
 }
 
+function getBookOnlineLink(req, res, next) {
+  const gpId = req.params.gpId;
+  const systemSupplier = cache.get(gpId).supplier_name;
+
+  switch (systemSupplier) {
+    case 'EMIS':
+      // eslint-disable-next-line no-param-reassign
+      req.gpDetails.bookOnlineLink = 'https://patient.emisaccess.co.uk/Account/Login';
+      break;
+    case 'Informatica':
```

Comment:
What is the difference between, for example, the Informatica case and EMIS(I)?

Also, I think we need to discuss with Joe what he expects when a direct link to booking is not possible and what the rational was for the decisions made in Choices which made them put a link to things other than a booking page.